e2etest/bgptests/bgp.go

```diff
@@ -662,6 +662,33 @@ var _ = ginkgo.Describe("BGP", func() {
 				return nil
 			}, 4*time.Minute, 1*time.Second).ShouldNot(HaveOccurred())
 
+			ginkgo.By("checking the sessions don't flap when changing the configuration")
+
+			previousNeighbors := map[string]frr.NeighborsMap{}
+			for _, c := range FRRContainers {
+				neighbors, err := frr.NeighborsInfoMap(c)
+				Expect(err).NotTo(HaveOccurred())
+				previousNeighbors[c.Name] = neighbors
+			}
+			ginkgo.By("creating another the service")
+			svc1, _ := testservice.CreateWithBackend(cs, testNamespace, "external-local-lb1", tweak)
+			testservice.Delete(cs, svc1)
+
+			Consistently(func() error {
+				for _, c := range FRRContainers {
+					neighbors, err := frr.NeighborsInfo(c)
+					Expect(err).NotTo(HaveOccurred())
+					Expect(neighbors).To(HaveLen(len(previousNeighbors[c.Name])))
+
+					for _, n := range neighbors {
+						previousDropped := previousNeighbors[c.Name][n.IP.String()].ConnectionsDropped
+						if n.ConnectionsDropped > previousDropped {
```

Comment:
Can you add in the commit one/two sentence doc about the timer? or link

Reply:
what timer?

Reply:
sorry I corrected timer, what is the doc for this ConnectionDropped

Reply:
it's a property of the output of `bgp show neighbor`. I can add a comment saying connectiondropped is the number of dropped connections, but feels a bit redundant.